lib/Dancer2/Core/Request.pm

```diff
@@ -742,6 +742,15 @@ If you need to access the body of the request, you have to use this accessor and
 should not try to read C<psgi.input> by hand. C<Dancer2::Core::Request>
 already did it for you and kept the raw body untouched in there.
 
+=method body_data
+
+Returns the body of the request in data form, making it possible to distinguish
+between C<body_parameters>, a representation of Web parameters
```

Comment:
Web parameters sounds weird to me and leaves wondering what they are. These come from a form or as "request" parameters, don't they?

Reply:
I was thinking request parameters might be a better fit here.

Reply:
I'm happy with any phrasing you prefer

Reply:
"request parameters" is better phrasing. Docs updated.